### ISSUE TITLE
Events with no `localEndTime` crash the `pruneEvents` scheduled job.

### ISSUE THREAD

**Issue by jhugman** [mozilla-mobile/prox-server#66]
```pre
ERROR:prox:Error running scheduled calendar fetch
Traceback (most recent call last):
  File "/home/prox-dev/prox-server/app/request_handler.py", line 250, in updateFromGcals
    pruneEvents()
  File "/home/prox-dev/prox-server/app/request_handler.py", line 229, in pruneEvents
    endTime = parser.parse(event.val().get("localEndTime"))
  File "/home/prox-dev/prox-server/env/local/lib/python2.7/site-packages/dateutil/parser.py", line 1168, in parse
    return DEFAULTPARSER.parse(timestr, **kwargs)
  File "/home/prox-dev/prox-server/env/local/lib/python2.7/site-packages/dateutil/parser.py", line 556, in parse
    res, skipped_tokens = self._parse(timestr, **kwargs)
  File "/home/prox-dev/prox-server/env/local/lib/python2.7/site-packages/dateutil/parser.py", line 675, in _parse
    l = _timelex.split(timestr)         # Splits the timestr into tokens
  File "/home/prox-dev/prox-server/env/local/lib/python2.7/site-packages/dateutil/parser.py", line 192, in split
    return list(cls(s))
  File "/home/prox-dev/prox-server/env/local/lib/python2.7/site-packages/dateutil/parser.py", line 61, in __init__
    '{itype}'.format(itype=instream.__class__.__name__))
TypeError: Parser must be a string or character stream, not NoneType

```

**Comment by liuche** [comment]
I'm pretty confused about this, because we should never write to the Firebase without an end time - if there's no end time, we just add an hour to the start time.

In any case, I'll push a commit to just delete items that don't have an end time.

**Comment by liuche** [comment]
Closed by #67
